instrumentserver/client/proxy.py

```diff
@@ -322,6 +322,14 @@ def _refreshProxySubmodules(self):
             else:
                 self.submodules[sn].update()
 
+    def __getattr__(self, item):
+        self.bp = self.cli.getBluePrint(self.remotePath)
+        self._getProxySubmodules()
+        try:
+            return super().__getattr__(item)
+        except AttributeError as e:
+            raise e
+
```

Comment:
This is interesting. Is that already enough? do we not need to also call ``self._getProxyParameters()``? And wouldn't calling ``self.update()`` be more consistent?

in the try block, why do you call ``super().__getattr__?`` i would have thought just ``getattr(self, item)`` should work.